src/TestUtils/src/Microsoft.Maui.IntegrationTests/BaseBuildTest.cs

```diff
@@ -13,8 +13,21 @@ public abstract class BaseBuildTest
 		public const string DotNetCurrent = "net9.0";
 		public const string DotNetPrevious = "net8.0";
 
-		public const string MauiVersionCurrent = "9.0.0-rc.1.24453.9"; // this should not be the same as the last release
-		public const string MauiVersionPrevious = "8.0.72"; // this should not be the same version as the default. aka: MicrosoftMauiPreviousDotNetReleasedVersion in eng/Versions.props
+		// Versions of .NET MAUI that are used when testing the <MauiVersion> property. These should preferrably
+		// different to the defaults in the SDKs such that the tests can test what would happen if the user puts
+		// some arbitrary number in <MauiVersion>. The actual numbers do not matter as much, as long as they trigger
+		// the MSBuild targets that would download some version that is only on nuget.org and not in the workload.
+		//
+		// MauiVersionCurrent: this should be the current .NET version of MAUI, but the latest released build.
+		// For example, if this branch is for .NET 9, then this must be a 9.0.x number. If the latest MAUI release
+		// is 9.0.100, then this should preferrable be some older build to make sure things work, like 9.0.30.
+		public const string MauiVersionCurrent = "9.0.0-rc.1.24453.9";
```

Comment:
```suggestion
		public const string MauiVersionCurrent = "9.0.20";
```

can this be 9.0.20 the current inserted workload we have?